packages/mds-ingest-service/repository/mappers/telemetry-mappers.ts

```diff
@@ -53,6 +58,9 @@ export const TelemetryDomainToEntityCreate = ModelMapper<
       heading,
       accuracy,
       altitude,
+      hdop,
```

Comment:
Are `hdop` and `accuracy` the same thing or could they potentially have different values?

Reply:
It seems like they could have different values given the [origin of accuracy in mds](https://github.com/openmobilityfoundation/mobility-data-specification/issues/260), I had the same question as I was going through this :)